lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -213,9 +213,9 @@ def add_address(post, options)
       def add_invoice(post, money, options)
         post[:payment][:amount_total] = amount(money)
         post[:payment][:currency_code] = (options[:currency] || currency(money))
-        post[:payment][:merchant_payment_code] = Digest::MD5.hexdigest(options[:order_id])
+        post[:payment][:merchant_payment_code] = Digest::MD5.hexdigest(order_id_override(options)) if order_id_override(options)
```

Comment:
Since the override method should return something everytime, we shouldn't need the `if` here